code/src/index.css

```diff
@@ -5,9 +5,169 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+  color: #ffffff;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* For the whole music albums */
+.music-container {
+  display: grid;
+  grid-template-columns: 100%;
+  margin: auto 0;
+  padding: 2%;
+}
+
+.theAlbums {
```

Comment:
The casing convention for CSS class names is typically kebab-case. Ultimately it doesn't matter which casing you go for, but probably best to be consistent throughout a project especially if other people will work on the same project. :)